readthedocs/core/utils/objects.py

```diff
@@ -0,0 +1,28 @@
+_DEFAULT = object()
```

Comment:
Probably worth a comment here about why we're doing this at the class level instead of in the function or similar.